### PR TITLE
attempt 1 css-brochure-form

### PR DESCRIPTION
**Pull request by Yi-Jacob**
Screen Shot 2022-01-27 at 8 54 24 AM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Jacob, your inputs are too wide and are not styled correctly to match the example, please revise and submit a new screenshot with your revision.

**Comment by Yi-Jacob** [comment]
Screen Shot 2022-01-27 at 3 14 16 PM

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Jacob! The borders on your input and select elements are different from the example, and your SUBMIT button is indented a bit from the right side of the select, where it should be resting against the right edge of the select element, please revise and submit a new screenshot with your revision.

**Comment by Yi-Jacob** [comment]
Screen Shot 2022-01-28 at 12 47 51 PM

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
+body {
+  font-family: sans-serif, "Brandon Grotesque";
+  font-style: normal;
+  font-weight: normal;
+  src: url("BrandonGrotesque-Regular.woff2") format("woff2");
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: These are only necessary for the @font-face rule, not using the font. You can just state the name of the fonts in the `fonts.css` file in the `font-family` property values to use it.

File: css-brochure-form/styles.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  margin: auto;
+}
+
+input,
+select {
+  margin: 6px;
+  border-radius: 5px;
+  width: 90%;
+  border-width: 50%;
+  margin-right: 50%;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: 😱 😱 😱 😱 

That's a **_lot_** of margin. Why is this necessary? There is likely a better way to accomplish this.

File: css-brochure-form/styles.css [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  padding: 5px;
+  align-content: center;
+  font-weight: lighter;
+}
+
+label {
+  font-weight: 400;
+  margin: 10px;
+  font-size: x-small;
+  margin-top: 75%;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: 😱 😱 😱 😱 

That's a **_lot_** of margin. Why is this necessary? There is likely a better way to accomplish this.

File: css-brochure-form/styles.css [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+input {
+  border-width: 0.5px;
+  border-color: rgb(202 205 204);
+  font-weight: lighter;
+}
+
+.button {
+  text-align: 80%;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: `text-align` doesn't accept percentage values, this is doing nothing.

File: css-brochure-form/styles.css [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+  text-transform: uppercase;
+  margin-top: 8px;
+  position: absolute;
+  right: 555px;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: 😱 😱 😱 😱 😱 😱 

You can do this in a better way.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by Yi-Jacob** [comment]
Screen Shot 2022-01-28 at 4 59 04 PM
i cleaned up some of the code

File: css-brochure-form/styles.css [hunk 6]
```diff
@@ -63,10 +58,9 @@ input {
 }
 
 .button {
-  text-align: 80%;
   text-transform: uppercase;
   margin-top: 8px;
-  position: absolute;
-  right: 555px;
+  position: relative;
+  left: 245px;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: This is just the same problem from a different side. You are pushing the element where you want it to be, which will break colossally when mobile responsiveness comes into play. You should use `text-align: right` to move the button to the right-hand side of the `div`, that will get you the same behavior in a way that works across all screen sizes.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by Yi-Jacob** [comment]
Screen Shot 2022-01-31 at 9 43 00 AM

sorry about that.

File: css-brochure-form/styles.css [hunk 7]
```diff
@@ -61,6 +61,6 @@ input {
   text-transform: uppercase;
   margin-top: 8px;
   position: relative;
-  left: 245px;
-  margin-left: 50px;
+  text-align: end;
+  padding-right: 3rem;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: Why is the padding necessary here? What happens if you leave it out?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by Yi-Jacob** [comment]
Screen Shot 2022-02-01 at 11 01 51 AM
the width of the buttons were not 100% so thats why i had padding but i changed it to 100% so now the padding was not needed

**Review by codingcodymiller — APPROVED** [review]
Nice work!!

**Pull request merged**